docs/src/pages/components/data-grid/getting-started/getting-started.md

```diff
@@ -126,52 +126,52 @@ If you have an enterprise grid running with an expired or missing license key, t
+| [Quick filter](/components/data-grid/filtering/#quick-filter)                             |    🚧     |              🚧               |                  🚧                   |
+| **Pagination**                                                                            |           |                               |                                       |
+| [Pagination](/components/data-grid/pagination/)                                           |    ✅     |              ✅               |                  ✅                   |
+| [Pagination > 100 rows per page](/components/data-grid/pagination/#paginate-gt-100-rows)  |    ❌     |              ✅               |                  ✅                   |
+| **Editing**                                                                               |           |                               |                                       |
+| [Row edition](/components/data-grid/editing/#row-editing)                                 |    🚧     |              🚧               |                  🚧                   |
+| [Cell editing](/components/data-grid/editing/#cell-editing)                               |    ✅     |              ✅               |                  ✅                   |
+| **Import & export**                                                                       |           |                               |                                       |
+| [CSV export](/components/data-grid/export/#csv-export)                                    |    ✅     |              ✅               |                  ✅                   |
+| [Print](/components/data-grid/export/#print)                                              |    🚧     |              🚧               |                  🚧                   |
+| [Excel export](/components/data-grid/export/#excel-export)                                |    ❌     |              ❌               |                  🚧                   |
+| [Clipboard](/components/data-grid/export/#clipboard)                                      |    ❌     |              🚧               |                  🚧                   |
+| **Rendering**                                                                             |           |                               |                                       |
+| [Column virtualization](/components/data-grid/virtualization/#column-virtualization)      |    ✅     |              ✅               |                  ✅                   |
+| [Row virtualization > 100 rows](/components/data-grid/virtualization/#row-virtualization) |    ❌     |              ✅               |                  ✅                   |
+| [Components](/components/data-grid/components/)                                           |    ✅     |              ✅               |                  ✅                   |
```

Comment:
"Components" as a title doesn't make sense. I don't know why I used this. How about?

```suggestion
| [Customization](/components/data-grid/components/)                                           |    ✅     |              ✅               |                  ✅                   |
```

Reply:
The link is to `/components/`. Maybe:
```suggestion
| [Customizable components](/components/data-grid/components/)                                           |    ✅     |              ✅               |                  ✅                   |
```

Reply:
Ok